project/Dependencies.scala

```diff
@@ -318,16 +235,27 @@ object Dependencies {
       exclude("com.google.guava", "guava-jdk5")
   )
 
+  /*
+  Used instead of `"org.lerch" % "s3fs" % s3fsV exclude("org.slf4j", "jcl-over-slf4j")`
+  org.lerch:s3fs:1.0.1 depends on a preview release of software.amazon.awssdk:s3.
+
+  Instead the code has been re-forked into this repo, just like many of the other FileSystemProvider extensions.
```

Comment:
😮

Reply:
Sub optimal, but I'm not aware of an actively maintained version of this code elsewhere.

Anyways, welcomed into the fold with all of the other NIO2 `FileSystemProvider` implementations already spread across this repo. [[1]](https://github.com/broadinstitute/cromwell/blob/fd5bdf873a9d9fae538c61e7e0945ce0d06a64b0/filesystems/oss/src/main/scala/cromwell/filesystems/oss/nio/OssStorageFileSystemProvider.scala#L20) [[2]](https://github.com/broadinstitute/cromwell/blob/develop/cloud-nio/cloud-nio-impl-drs/src/main/scala/cloud/nio/impl/drs/DrsCloudNioFileSystemProvider.scala#L15-L18) [[3]](https://github.com/broadinstitute/cromwell/blob/develop/cloud-nio/cloud-nio-impl-ftp/src/main/scala/cloud/nio/impl/ftp/FtpCloudNioFileSystemProvider.scala#L14)